modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase

```diff
@@ -117,6 +117,78 @@ definition {
 		}
 	}
 
+	@description = "LPS-168228 - Verify it's possible to add many custom fields in the custom layout"
+	@priority = "4"
+	test CanAddManyFieldsOnLayout {
+		task ("Given an object with 8 fields is created") {
```

Comment:
Update this task block to reflect your actual number of fields

Reply:
Done!